about.html

```diff
@@ -0,0 +1,118 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="styles.css" rel="stylesheet" type="text/css">
+    <script src="https://kit.fontawesome.com/b33cf67da8.js" crossorigin="anonymous"></script>
+    <script src="./profiles.js" defer></script>
+    <script src="./main.js" defer></script>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link href="https://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Lato&display=swap" rel="stylesheet">
+    <title>Gen Mex Concert 2023 Amazon Originals</title>
+</head>
+<body>
+    <div class="social-media-desktop-about display-none">
+        <div class="first-container-desktop-about">
+        <i class="fa-brands fa-facebook-f"></i>
+        <i class="fa-brands fa-twitter"></i>
+        <p>English</p>
+        <p>My Page</p>
+        <p>Logout</p>
+        </div>
+    </div>
+    <div class="nav-bar-about display-none">
+        <div class="nav-bar-container">
+        <a href="./index.html" class="logo"><img src="images/genmexlogo.jpg" alt="Gen Mex Concert Logo"></a>
+        <nav class="nav-menu-about">
+        <ul class="nav-options-about">
+            <li class="nav-link-about"><a class="link-menu-about" href="./about.html">About</a></li>
+            <li class="nav-link-about"><a class="link-menu-about" href="#">Program</a></li>
+            <li class="nav-link-about"><a class="link-menu-about" href="#">Join</a></li>
+            <li class="nav-link-about"><a class="link-menu-about" href="#">Sponsor</a></li>
+            <li class="nav-link-about"><a class="link-menu-about" href="#">News</a></li>
+            <li class="nav-link-about"><a id="gmcampaign" class="link-menu-about" href="#">GM Campaign</a></li>
+        </ul>
+        </nav>
+        </div>
+    </div>
+    <header class="principal-info-about">
+        <div class="mobile-bar display-none">
+            <nav class="mobile-navmenu-about">
+            <ul class="mobile-nav-options">
+                <li class="mobile-nav-link"><i class="fa-solid fa-xmark"></i></li>
```

Comment:
-[Optional] Kindly it would be better if you move the close `X` button to the top right for having better user experience👍
you can use `display: absolute` to achieve this 👇
```
.fa-solid{
    position: absolute;
    color: white;
    top: -10px;
    right: -10px;
}
```